doc: Bluetooth: Expand BabbleSim section (#14470)
zephyrproject-rtos/zephyr: doc/guides/bluetooth/bluetooth-tools.rst

********************************

It's possible to run Bluetooth applications using either the :ref:`QEMU
emulator<application_run_qemu>` or :ref:`Native POSIX <native_posix>`.

> In order to do so, a Bluetooth controller needs to be exported from

Reviewer: ```suggestion
In either case, a Bluetooth controller needs to be exported from
```
